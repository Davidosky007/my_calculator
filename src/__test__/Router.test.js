/* eslint-disable linebreak-style */
import { render, screen } from '@testing-library/react';
import { create } from 'react-test-renderer';
import '@testing-library/jest-dom';
import AppRoute from '../components/Router';

describe('test for AppRouter component', () => {
  test('it renders Home component with text Math Magician', () => {
    render(<AppRoute/>);
    expect(screen.getByText('Math Magician')).toBeInTheDocument();
  }); test('renders AppRouter component properly', () => {
    const home = create(<AppRoute/>).toJSON();
    expect(home).toMatchSnapshot();
  });
});
