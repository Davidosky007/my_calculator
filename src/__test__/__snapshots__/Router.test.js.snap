// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for AppRouter component renders AppRouter component properly 1`] = `
Array [
  <header
    className="main_nav"
  >
    <div>
      <h1>
        Math Magician
      </h1>
    </div>
    <nav>
      <ul
        className="nav_list"
      >
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            href="/qoutes"
            onClick={[Function]}
          >
            Qoutes
          </a>
        </li>
      </ul>
    </nav>
  </header>,
  <div
    className="main_home"
  >
    <h4>
      Welcome to our page
    </h4>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
    </p>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
    </p>
  </div>,
]
`;
